testing/src/test/java/org/motechproject/nms/testing/it/kilkari/MctsBeneficiaryImportServiceBundleIT.java

```diff
@@ -1261,14 +1241,13 @@ public void verifyFT526() throws Exception {
     }
 
     /*
-     * To verify mother MCTS upload is rejected when mandatory parameter state is having invalid value. 
+     * To verify mother MCTS upload is rejected with invalid district id
      */
     @Test
     public void verifyFT527() throws Exception {
-        DateTime lmp = DateTime.now().minusDays(30);
-        String lmpString = getDateString(lmp);
-        Reader reader = createMotherDataReader("21\t6\t\t\t\t\t1234567890\tShanti Ekka\t9439986187\t\t" +
-                lmpString + "\t\t\t");
+        String dobString = getDateString(DateTime.now().minusDays(30));
+        Reader reader = createChildDataReader("21\t6\t\t\t\t\t1234567890\tBaby1 of Lilima Kua\t\t9439986187\t"
```

Comment:
aaccckk... these are the kinda things I fixed all weekend :crying_cat_face: . fixed  now